ucp/core.py

```diff
@@ -397,9 +397,9 @@ def continuous_ucx_progress(self, event_loop=None):
         ----------
         event_loop: asyncio.event_loop, optional
             The event loop to evoke UCX progress. If None,
-            `asyncio.get_event_loop()` is used.
+            `ucp.utils.get_event_loop()` is used.
         """
-        loop = event_loop if event_loop is not None else asyncio.get_event_loop()
+        loop = event_loop if event_loop is not None else get_event_loop()
```

Comment:
Slightly different style to the previous code, but probably `event_loop or get_event_loop()` here too.